testworld/payloads.go

```diff
@@ -24,6 +24,22 @@ func defaultPOPayload(collaborators []string) map[string]interface{} {
 		"collaborators": collaborators,
 	}
 }
+
+func defaultEntityPayload(collaborators []string) map[string]interface{} {
+	return map[string]interface{}{
+		"data": map[string]interface{}{
+			"identity":   "0x36b264199f891762BA4B2A6574b06c2868252a8A",
```

Comment:
We need to make sure that the identity in the entity is a real value associated to a real identity. As this is testworld we should pass the identity as part of the method signature